README.md

```diff
@@ -144,7 +144,7 @@ This will cut a tag version, builds the gem, and pushes the gem up to Rubygems
 
 Note: You may need permission to push a gem up to Rubygems!
 You will first need to create an account on rubygems.org.
-Once you have an account, bug Shane or Matt to [add you as an owner](http://guides.rubygems.org/command-reference/#gem-owner) to pushmi_pullyu Rubygem. Once you are an owner you should be able to push new versions of pushmi_pullyu up to Rubygems.
+Once you have an account, bug Tricia, Omar, or Henry to [add you as an owner](http://guides.rubygems.org/command-reference/#gem-owner) to pushmi_pullyu Rubygem. Once you are an owner you should be able to push new versions of pushmi_pullyu up to Rubygems.
```

Comment:
What do you think if we mention the usernames for the people to bug ?

```suggestion
Once you have an account, bug @pgwillia (Tricia Jenkins), @lagoan (Omar Rodriguez-Arenas), or @henryzhang87 (Henry Zhang) to [add you as an owner](http://guides.rubygems.org/command-reference/#gem-owner) to pushmi_pullyu Rubygem. Once you are an owner you should be able to push new versions of pushmi_pullyu up to Rubygems
```

We did the same for the [update on the DI internal documentation](https://github.com/ualbertalib/di_internal/blob/1440b7375d899d6f0d11a18c5c0a19367bbb2ea4/DI_Dev_Overview.md?plain=1#L198-L206).

Reply:
> What do you think if we mention the usernames for the people to bug ?
> 
> We did the same for the [update on the DI internal documentation](https://github.com/ualbertalib/di_internal/blob/1440b7375d899d6f0d11a18c5c0a19367bbb2ea4/DI_Dev_Overview.md?plain=1#L198-L206).

Good idea!